client/src/components/RecipeNavigator.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const RecipeList = ({ recipes, onClick }) => {
+    if (recipes && Array.isArray(recipes)) {
+        return recipes.map((recipe,index) => <li onClick={() => onClick(index)}>{recipe}</li>);
```

Comment:
Don't forget to give each li a unique key attribute
```suggestion
        return recipes.map((recipe,index) => <li key={`whatever-id-you-pick`}onClick={() => onClick(index)}>{recipe}</li>);
```

Reply:
👍